content/outcomes/cloud-native-development/specific__batch_processing.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Batch Processing
+weight: 150
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+There are numerous ways to schedule and run batch jobs with Spring.
```

Comment:
For credibility we should consider describing and referencing some non-Spring alternatives.

Reply:
Done! Good point.

6d67064b950fb78a2bc745f6688ae5d2b7d5f6df